torch/csrc/jit/codegen/cuda/lower_utils.cpp

```diff
@@ -277,6 +337,195 @@ std::pair<kir::ForLoop*, int64_t> getAllocPoint(
 }
 
 } // namespace loop_utils
+
+namespace kir_utils {
```

Comment:
Do we really need to have yet another `utils` name space? There are `ir_utils`, `loop_utils`, `scope_utils`, etc, and it's not really clear to me which one to use for which functions.